File: java/06-io-operations/ConsoleInput.java
import java.util.Scanner;

public class ConsoleInput {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.println("What is your name?");
        String name = scanner.nextLine();

        System.out.println("What year were you born?");
        int birthYear = scanner.nextInt();

        int age = 2026 - birthYear;
        System.out.printf("Hi %s, you are about %d this year.%n", name, age);

        scanner.close();
    }
}

File: pascal/06-io-operations/console_input.pas
program ConsoleInput;

var
    userName: string;
    age: Integer;

begin
    Write('Enter your name: ');
    ReadLn(userName);

    Write('Enter your age: ');
    ReadLn(age);

    WriteLn('Hello, ', userName, '!');
    WriteLn('Next year you will be ', age + 1, ' years old.');
end.
